BLSQ/iaso
# IA-2060:  Registry add a list of children org units

In the registry, the map is very useful, but it would be good to have a list of org units too. 

Needed columns: name, type, and an icon to see if it is has a shape or location. 
Consequently, beware that the list should include ALL children, not only the ones that have a shape or location

Related JIRA tickets : IA-2060

## Self proofreading checklist


## Changes

- split calls for map and list
- dedicated component for tabs and display list or map
- routes, order, pagination, deep link

## How to test

Open registry for an org units with direct children.
You should be able to see a list of those children.
Play with the pagination, order columns, test deep link

## Print screen / video

https://user-images.githubusercontent.com/12494624/233580658-398c0955-a715-431c-8eb6-7e244bd43e6f.mov
